packages/react-components/src/components/VideoGallery.tsx

```diff
@@ -259,28 +259,30 @@ export const VideoGallery = (props: VideoGalleryProps): JSX.Element => {
       onCreateLocalStreamView && onCreateLocalStreamView(localVideoViewOptions);
     }
     return (
-      <VideoTile
-        key={localParticipant.userId}
-        userId={localParticipant.userId}
-        renderElement={
-          localVideoStream?.renderElement ? (
-            <>
-              {
-                /* @conditional-compile-remove(call-with-chat-composite) @conditional-compile-remove(local-camera-switcher) */
-                localCameraCycleButton(localVideoCameraCycleButtonProps)
-              }
-              <StreamMedia videoStreamElement={localVideoStream.renderElement} />
-            </>
-          ) : undefined
-        }
-        showLabel={!(shouldFloatLocalVideo && isNarrow)}
-        displayName={isNarrow ? '' : strings.localVideoLabel}
-        initialsName={localParticipant.displayName}
-        styles={localVideoTileStylesThemed}
-        onRenderPlaceholder={onRenderAvatar}
-        isMuted={localParticipant.isMuted}
-        showMuteIndicator={showMuteIndicator}
-      />
+      <Stack tabIndex={0}>
```

Comment:
I think at all times when something has focus there needs to be a change in this component that indicates this (typically a border being applied/made thicker). I can't find where something defines this but typically you are always able to spot when something has focus -- should we apply a border of some kind to this Stack (PIP) when it has focus?

Reply:
I was wondering this as well, I will get in touch with Alex P about it since it should probably have different display than a button since it has a hidden drop down? or should we just add the button selections to it?